airflow/config_templates/default_airflow.cfg

```diff
@@ -349,6 +349,7 @@ smtp_ssl = False
 smtp_port = 25
 smtp_mail_from = airflow@example.com
 
+smtp_mime_from = airflow
```

Comment:
Can you show how the message looks now? I think it will be better if this field is empty in default configuration.

Reply:
@mik-laj  i agree. Applied a fix